src/composables/useEthers.ts

```diff
@@ -106,9 +104,9 @@ export default function useEthers() {
       if (receipt != null) {
         finalizeTransaction(txHash, 'tx', receipt);
       }
-      callbacks.onTxConfirmed(receipt);
-      if (shouldRefetchBalances && !supportsBlocknative.value) {
```

Comment:
`!supportsBlocknative` is a check to prevent refetching of the balances, we really need to do it here too to ensure we know the balance after the post-action delay we have on networks like Polygon. It's possible we can just remove blocknative in the future, and do all the fetching here instead.